src/components/balances/TokenListSelect/index.tsx

```diff
@@ -10,6 +9,7 @@ import { OnboardingTooltip } from '@/components/common/OnboardingTooltip'
 import Track from '@/components/common/Track'
 import { ASSETS_EVENTS, trackEvent } from '@/services/analytics'
 import { HelpCenterArticle } from '@/config/constants'
+import { useHasFeature } from '@/hooks/useChains'
```

Comment:
There are still a few other instances where we have not replaced `hasFeature` from what I can tell.

Note: this is not related to this line.

Reply:
I noticed that too but afaik the places where we still use `hasFeature` either need the currentChain dependency for other things or its somewhere where we can't use hooks.